### PR TITLE
fix: use dimension data on 1.16.2

### PR DESCRIPTION
**Pull request by zardoy**
dimension packet is available starting from 1.16.2

finishing https://github.com/PrismarineJS/minecraft-data/pull/877

### CODE REVIEW

**Comment by zardoy** [comment]
tests are passing locally and I can't scroll to the error on the github...

File: lib/plugins/blocks.js [hunk 1]
```diff
@@ -261,7 +261,7 @@ function inject (bot, { version, storageBuilder, hideErrors }) {
       bot.world.setColumn(packet.chunkX, packet.chunkZ, column)
     }
 
-    if (bot.supportFeature('dimensionDataIsAvailable')) {
+    if (bot.supportFeature('newLightingDataFormat')) {
```
Comment (rom1504) [inline lib/plugins/blocks.js:264]: this is for 1.17 https://github.com/PrismarineJS/minecraft-data/blob/8a8c811c71790d75ec405429e5b50343cb77f88b/data/pc/common/features.json#L785

that contradicts the PR description

which is right?

Comment (zardoy) [reply lib/plugins/blocks.js:264]: > that contradicts the PR description

Didn't say anything about this change in the pr description, but this change is correct: `loadParsedLight` is available starting from 1.17 (new lighting format)

**Comment by zardoy** [comment]
What user input does it wait for?

**Review by extremeheat — APPROVED** [review]

**Comment by zardoy** [comment]
@rom1504 Thank you

**Pull request merged**